contracts/ts/deploy.ts

```diff
@@ -339,103 +376,20 @@ const deployMaci = async (
     }
 }
 
-const main = async () => {
-    let accounts
-    if (config.env === 'local-dev' || config.env === 'test') {
-        accounts = genTestAccounts(1)
-    } else {
-        accounts = genAccounts()
-    }
-    const admin = accounts[0]
-
-    console.log('Using account', admin.address)
-
-    const parser = new argparse.ArgumentParser({
-        description: 'Deploy all contracts to an Ethereum network of your choice'
-    })
-
-    parser.addArgument(
-        ['-o', '--output'],
-        {
-            help: 'The filepath to save the addresses of the deployed contracts',
-            required: true
-        }
-    )
-
-    parser.addArgument(
-        ['-s', '--signUpToken'],
-        {
-            help: 'The address of the signup token (e.g. POAP)',
-            required: false
-        }
-    )
-
-    parser.addArgument(
-        ['-p', '--initialVoiceCreditProxy'],
-        {
-            help: 'The address of the contract which provides the initial voice credit balance',
-            required: false
-        }
-    )
-
-    parser.addArgument(
-        ['-v', '--verifier'],
-        {
-            help: 'The address of the contract which verifies zk-SNARK proofs',
-            required: true
-        }
-    )
-
-    const args = parser.parseArgs()
-    const outputAddressFile = args.output
-    const signUpToken = args.signUpToken
-    const initialVoiceCreditProxy = args.initialVoiceCreditProxy
-    const verifier = args.verifier
-
-    const deployer = genDeployer(admin.privateKey)
-
-    let signUpTokenAddress
-    if (signUpToken) {
-        signUpTokenAddress = signUpToken
-    } else {
-        const signUpTokenContract = await deploySignupToken(deployer)
-        signUpTokenAddress = signUpTokenContract.address
-    }
-
-    const signUpTokenGatekeeperContract = await deploySignupTokenGatekeeper(
-        deployer,
-        signUpTokenAddress,
-    )
-
-    let initialVoiceCreditBalanceAddress
-    if (initialVoiceCreditProxy) {
-        initialVoiceCreditBalanceAddress = initialVoiceCreditProxy
-    } else {
-        const initialVoiceCreditProxyContract = await deployConstantInitialVoiceCreditProxy(
-            deployer,
-            config.maci.initialVoiceCreditBalance,
-        )
-        initialVoiceCreditBalanceAddress = initialVoiceCreditProxyContract.address
-    }
-
-    const {
-        maciContract,
-        vkRegistryContract,
-        stateAqContract,
-        pptContract,
-    } = await deployMaci(
-        deployer,
-        signUpTokenGatekeeperContract.address,
-        initialVoiceCreditBalanceAddress,
-        verifier,
-    )
-
+const writeContractAddress = (
```

Comment:
Since we are getting rid of the main function for the module ,I moved the file writing logic to a separate function that can be used in the tests later